loader.js

```diff
@@ -113,23 +113,20 @@ define([
       });
     }
 
-    // install optional plugins, summary widget is handled separately as it was added long ago
+    // install optional plugins
     if (config.plugins) {
-      if (
-        config.plugins.summaryWidgets === true ||
-        config.plugins.summaryWidgets?.enabled === true
-      ) {
-        openmct.install(openmct.plugins.SummaryWidget());
-      }
-
       Object.entries(config.plugins).forEach(([plugin, pluginConfig]) => {
-        const pluginExists = openmct.plugins[plugin] || openmct.plugins.example[plugin];
+        const examplePluginExists = openmct.plugins.example[plugin];
+        const pluginExists = openmct.plugins[plugin] || examplePluginExists;
         const pluginEnabled = pluginConfig?.enabled;
```

Comment:
should we follow the previous format summary widgets followed, or just require an enabled flag now?
```
config.plugins.summaryWidgets === true ||
config.plugins.summaryWidgets?.enabled === true
```